### PR TITLE
Fix #2673: Remove event_logger_java_proto_lite export

### LINKED ISSUE #2673: Remove :event_logger_java_proto_lite export from the model library
This task is part of the build system migration from Gradle to Bazel. A prerequesite before working on this issue is that you are able to build Oppia using Bazel. Check out [Oppia Bazel guide](https://github.com/oppia/oppia-android/wiki/Oppia-Bazel-Guide) for Bazel terminology and more on what Bazel does.

## 1. Remove the export

In 

`/oppia-android/model/BUILD.bazel`

Remove :event_logger_java_proto_lite from the list of exports in the model library

```BUILD
android_library(
    name = "model",
    visibility = ["//visibility:public"],
    exports = [
        ":event_logger_java_proto_lite", # <-- Remove this line
        ":exploration_java_proto_lite",
        ":interaction_object_java_proto_lite",
        ":onboarding_java_proto_lite",
        ":profile_java_proto_lite",
        ":question_java_proto_lite",
        ":subtitled_html_java_proto_lite",
        ":subtitled_unicode_java_proto_lite",
        ":thumbnail_java_proto_lite",
        ":topic_java_proto_lite",
        ":translation_java_proto_lite",
        ":voiceover_java_proto_lite",
    ],
)
```

## 2. Add public visibility to the java_lite_proto_library

Add 

`    visibility = ["//visibility:public"],`

to the java_lite_proto_library that has the name event_logger_java_proto_lite.

## 3. Build the app using Bazel

Libraries that are indirectly depending on `event_logger_java_proto_lite` will now fail to build.  Try to build the app with Bazel and add the deps dependency

`//model:event_logger_java_proto_lite,`

to all the libraries that are now failing to build. You will have to look around for BUILD.bazel files that have libraries that require this replaced dependency.

These libraries will already have the //model dependency and since we removed the :event_logger_java_proto_lite from the model library itself, they will need to add the newly available public library directly as a dependency. (Not all libraries with the //model dependency may require the newly available library)

Once the app builds, you're done!

### PR DESCRIPTION
**Pull request by Sparsh1212**
## Explanation

Fixes #2673: Remove :event_logger_java_proto_lite export from the model library. 
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Sparsh1212** [comment]
@fsharpasharp @rt4914 PTAL.

File: model/BUILD.bazel [hunk 1]
```diff
@@ -47,6 +47,7 @@ proto_library(
 java_lite_proto_library(
     name = "event_logger_java_proto_lite",
     deps = [":event_logger_proto"],
+    visibility = ["//visibility:public"],
```
Comment (vinitamurthi) [inline model/BUILD.bazel:59]: Any reason why we are setting visibility to public?

Comment (BenHenning) [reply model/BUILD.bazel:59]: This is actually the intention of #2673. We want to move away from omni model group libraries and to depending on specific targets.

**Comment by BenHenning** [comment]
We need to see if all of the corresponding Bazel targets build after this change, so updating the branch to re-trigger the build. It will probably take a while (~8 hours) to complete.

**Review by fsharpasharp — APPROVED** [review]
Looks good.

**Review by BenHenning — APPROVED** [review]
Thanks @Sparsh1212. This also LGTM.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Ah, actually @Sparsh1212 please address the conflicts & then I'll take another pass. It seems like there are some conflicts that need to be addressed before this can be merged.

**Comment by Sparsh1212** [comment]
@BenHenning I have resolved the merge conflicts, updated the branch with latest develop branch and the tests have also passed. PTAL

**Comment by Sparsh1212** [comment]
@rt4914 @BenHenning Please take a look at this. Also, please let me know if some discussion is going on all the similar "Remove export" issues. So shall I work on some different issues till then if the reviewing will take some more time? like as pointed out in this [comment](https://github.com/oppia/oppia-android/pull/2700#issuecomment-780822120)

**Review by BenHenning — APPROVED** [review]
Apologies @Sparsh1212 I missed your earlier comment. This LGTM--thanks!

**Pull request merged**